### PR TITLE
[USER ACTION] Change in the README

### PR DESCRIPTION
**Pull request by shekhar4nov**
I changed a word in the README.

### CODE REVIEW

**Comment by johnjacobkenny** [comment]
@shekhar4nov You will need to get the latest version of the project and make a PR from that :smile:

**Comment by shekhar4nov** [comment]
Ok, I am trying again

**Comment by shekhar4nov** [comment]
I made changes again......please check

**Comment by johnjacobkenny** [comment]
@shekhar4nov I think you might need to take latest and retry again

**Comment by shekhar4nov** [comment]
> @shekhar4nov I think you might need to take latest and retry again

I forked it, did changes and then clicked on commit changes....ok again I do

**Comment by johnjacobkenny** [comment]
@Sheetal777 is there any docs on how to get project synced up and then make the change for PR

**Comment by arthurbuhl** [comment]
@johnjacobkenny Its mentioned in the Documentation (#41) already.

**Comment by Sheetal777** [comment]
Please read the documentation step by step and then send the PR. @shekhar4nov , it's showing conflicting files so click on "Resolve issue" and get back to us. 👍

File: README.md [hunk 1]
```diff
@@ -1,7 +1,13 @@
+main
+# Twindle
+## What is Twindle?
+    Twindle is an open-source project to convert Twitter thread into a ePub, PDf document to allow developers and engineers to read through the interested thread information on the digital devices or print a hard copy.
+
 # Twindle 📖 
 [![Open Source Love](https://firstcontributions.github.io/open-source-badges/badges/open-source-v1/open-source.svg)](https://github.com/firstcontributions/open-source-badges)  [![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
 [![MIT Licence](https://badges.frapsoft.com/os/mit/mit.svg?v=103)](https://opensource.org/licenses/mit-license.php)
 ![image](https://visitor-badge.glitch.me/badge?page_id=Twindle-co.twindle)
+ main
```
Comment (arthurbuhl) [inline README.md:10]: I think you have added this text `main` here by mistake.

**Review by shekhar4nov — COMMENTED** [review]
I deleted that main. So, please review.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+main
```
Comment (arthurbuhl) [inline README.md:1]: There is one more `main` left here.

**Review by shekhar4nov — COMMENTED** [review]
I delete main. Please review

**Comment by johnjacobkenny** [comment]
@shekhar4nov I think you will need to get the latest version of readme in order to proceed

**Comment by proful** [comment]
close this pr if you are not planning to fix it. @shekhar4nov

**Pull request closed without merging**